docs/src/pages/api-reference/react.mdx

```diff
@@ -52,6 +52,13 @@ export const OurUploadButton = () => (
 | onUploadError          | function | No                      |                 | callback function when upload fails                                                 |
 | onUploadProgress       | function | No                      | Added in `v5.1` | callback function for upload progress                                               |
 | onUploadBegin          | function | No                      | Added in `v5.4` | callback function for upload begin                                                  |
+| config                 | Config   | No                      | Added in `v5.7` | object to pass additional configuration for button                                  |
+
+Config object
+
+| Attribute    | Type    | Required | Notes           | Description                                    |
+| :----------- | :------ | :------- | :-------------- | :--------------------------------------------- |
+| disablePaste | boolean | No       | Added in `v5.7` | disables ability to paste files from clipboard |
```

Comment:
imo options should be positive, so this should probably be something along the lines of `appendOnPaste` (naming is hard) with `true` as default (or maybe even `false` now and we can change default to true in next major, depending on if you see this as a breaking change or a new feature)

negated options are often just harder to parse

Reply:
Fixed